config/config.go

```diff
@@ -434,6 +435,7 @@ func NewConfig() *Config {
 	// Swarm:
 	flag.BoolVar(&cfg.EnableSwarm, "enable-swarm", false, "enable swarm communication between nodes in a skipper fleet")
 	flag.Var(cfg.SwarmRedisURLs, "swarm-redis-urls", "Redis URLs as comma separated list, used for building a swarm, for example in redis based cluster ratelimits.\nUse "+redisPasswordEnv+" environment variable or 'swarm-redis-password' key in config file to set redis password")
+	flag.StringVar(&cfg.SwarmRedisHashAlgorithm, "swarm-redis-hash-algorithm", "", "sets hash algorithm to be used in redis ring client to find the shard <jump|mpchash|rendezvousVnodes>, defaults to library default rendezvous")
```

Comment:
I think you may put `rendezvous` as a default value and then help message would mention it automatically so `, defaults to library default rendezvous` could be omitted.

Reply:
Also `rendezvous` should be be in the list of allowed values

Reply:
Although redis client library may change the default algorithm in the future.
Maybe we can have two options then: 
- when not set uses redis client library default whatever it is
- `rendezvous` uses rendezvous implementation (we would probably need to define a compatible wrapper)

Reply:
change as suggested